lib/pr_checker.js

```diff
@@ -172,6 +203,13 @@ class PRChecker {
     for (const [type, ci] of ciMap) {
       const name = CI_TYPES.get(type).name;
       logger.info(`Last ${name} CI on ${ci.date}: ${ci.link}`);
+      commits.forEach((c) => {
+        const commit = c.commit;
+        console.log(commit);
+        if (ci.date < commit.committedDate) {
+          logger.warn(`commit(${commit.message}) after Last ${name} CI`);
```

Comment:
It might be more helpful if this took a format such as:

```
[WARN] Commits pushed after last CI run:
[WARN] fixup: adjust spelling
[WARN] doc: add documentation of changes
```

Basically an intro and then just output the commit title (first line).